coalib/output/Logging.py

```diff
@@ -15,18 +15,24 @@ def configure_logging():
     logging.config.dictConfig({
         'version': 1,
         'handlers': {
-            'colored': {
+            'console-logger': {
                 'class': 'logging.StreamHandler',
-                'formatter': 'colored',
+                'formatter': 'color',
                 'stream': sys.stderr
+            },
+            'file-logger': {
+                'class': 'logging.FileHandler',
+                'formatter': 'no-color',
+                'filename': 'coala-log',
+                'mode': 'w+'
```

Comment:
I am having a bit of difficulty doing this. I can't understand how the arg `-L ` works.